list/list.html

```diff
@@ -0,0 +1,316 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/f8241a6d57.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="../base.css" />
+    <link rel="stylesheet" href="../components.css" />
+    <link rel="stylesheet" href="list.css" />
+    <title>Avatar</title>
+  </head>
+  <body>
+    <div class="nav-bar pad-1-3 flex-align">
+      <div class="left-nav mar-r-auto">
+        <a class="fnt-1-5 fnt-w-900" href="#">Pinakin</a>
+      </div>
+      <div class="right-nav">
+        <ul class="right-nav-list gap-1-5">
+          <li class="home">
+            <a href="../index.html"><i class="fnt-1-5 fas fa-home"></i></a>
+          </li>
+          <li class="github">
+            <a href="https://github.com/ashishnnnnn"
```

Comment:
I  think you should use target="_blank" for external links.

Reply:
Done